style.css

```diff
@@ -390,25 +355,117 @@ hr {
   padding: 8px;
 }
 
+#submit {
+  width: 110px;
+  font-size: 17px;
+  line-height: 24px;
+  color: #6070ff;
+  background: #fff;
+  border: 1px solid #7f8cff;
+  border-radius: 8px;
+}
+
+/*  Navbar Hamburger Css ::clicked */
+.click {
+  background: #6070ff;
+  position: fixed;
+  backdrop-filter: blur(5px);
+  margin: 0;
+  padding: 45px;
+  transition: all 0.5s;
+}
```

Comment:
While you did implement the previous reviewer's requirement, you didn't do so properly. Please make sure you remove the `backdrop-filter` rule and add an `opacity` rule and set it to 0.9 to match the Figma design.